src/cgame/cg_rocket_datasource.cpp

```diff
@@ -765,8 +765,14 @@ static void AddToAlOutputs( char *name )
 
 void CG_Rocket_SetAlOutputsOutput( const char*, int index )
 {
+	const char *device = "";
+	if (index >= -1 && index < rocketInfo.data.alOutputsCount )
+	{
+		device = rocketInfo.data.alOutputs[ index ];
```

Comment:
I think `index >= -1` is not what you wanted

Reply:
Indeed